Competition2017/src/competition/operator_interface/OperatorInterface.java

```diff
@@ -31,7 +31,8 @@ public OperatorInterface(WPIFactory factory, RobotAssertionManager assertionMana
         
         leftJoystick = factory.getJoystick(1);
         rightJoystick = factory.getJoystick(2);
-     
+        operatorPanelJoystick = factory.getJoystick(3);
```

Comment:
I suspect there's an existing joystick for the panel; this looks like a bad merge.